charts/stable/dnscrypt-proxy/values.yaml

```diff
@@ -0,0 +1,95 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: klutchell/dnscrypt-proxy
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: 2.1.1
```

Comment:
```suggestion
  tag: 
```

Reply:
Ok. I use everywhere I can k8s-at-home's charts, and I always saw fixed tag version, so I was thinking you prefer regular chart update but being sure nothing will break after a reboot for example and the download of a unexpected new version

Reply:
Sorry, I understood. It's already set to the Chart.yaml appVersion